buzzmobile/sense/inputer/inputer.py

```diff
@@ -10,7 +10,7 @@ def input_node():
     # Publish a nonsense initial value to make sure subscribers don't explode
     pub.publish("")
     while not rospy.is_shutdown():
-        new_dst = raw_input("Dest >> ")
+        new_dst = input("Dest >> ")
```

Comment:
well, in python3, there is no `raw_input`, so if by advisable, you mean 'the only way', then yes.

And running the node runs the python3 version.